core/src/mindustry/world/Block.java

```diff
@@ -671,6 +671,14 @@ public String getDisplayName(Tile tile){
         return localizedName;
     }
 
+    public String getModName(String prefix){
```

Comment:
These method should be in `Content`, since they're not block-specific.

Reply:
youre right ill move it now